README.md

```diff
@@ -949,15 +950,223 @@ To make writing a SPA easier, a number of frameworks have been developed. The mo
 - [Vue.js](https://vuejs.org/)
 - [Angular](https://angular.io/)
 
-They have their strengths and weaknesses which are summarized in the [NLeSC guide](https://guide.esciencecenter.nl/best_practices/language_guides/JavaScript.html#frameworks).
+They have their strengths and weaknesses which are summarized in the [NLeSC guide](https://guide.esciencecenter.nl/#/best_practices/language_guides/javascript?id=frameworks).
 
 <!-- Bubble below might need to be Newton-Raphson? -->
 
-For Bubble I picked React as it is light and functional, because I like the small api footprint and the functional programming paradigm.
+For NewtonRaphson web application I picked React as it is light and functional, because I like the small api footprint and the functional programming paradigm.
```

Comment:
```suggestion
For Newton-Raphson web application I picked React as it is light and functional, because I like the small API footprint and the functional programming paradigm.
```